src/rules/arrayTypeRule.ts

```diff
@@ -14,18 +15,21 @@ export class Rule extends Lint.Rules.AbstractRule {
             One of the following arguments must be provided:
 
             * \`"${OPTION_ARRAY}"\` enforces use of 'T[]'.
-            * \`"${OPTION_GENERIC}"\` enforces use of 'Array<T>'.`,
+            * \`"${OPTION_GENERIC}"\` enforces use of 'Array<T>'.
+            * \`"${OPTION_ARRAY_SIMPLE}"\` enforces use of 'T[]' if 'T' is a simple type (primitive or type reference).`,
```

Comment:
nit: should these single quotes be backticks? for when this doc is presented in markdown?